unit/util/sharing_map.cpp

```diff
@@ -378,6 +378,9 @@ TEST_CASE("Sharing map views and iteration", "[core][util]")
   SECTION("Iterate")
   {
     sharing_map_standardt sm;
+
+    sm.iterate([](const irep_idt &key, const std::string &value) {});
```

Comment:
So ... what does this test other than that it compiles?

Reply:
This just checks that calling `iterate()` on an empty map doesn't violate an internal assertion of the map.